src/db.rs

```diff
@@ -1,3 +1,5 @@
+use std::sync::Arc;
```

Comment:
Seems like your IDE is fighting with us. Can you run `cargo fmt` and `cargo clippy --all-targets --all-features` to make sure everything is clean?